src/webviews/webview-side/ipywidgets/common/helper.ts

```diff
@@ -36,11 +41,17 @@ export async function isCDNReachable() {
 }
 
 async function isWebSiteReachable(url: string, signal: AbortSignal) {
+    let retries = 0;
     try {
-        const response = await fetch(url, { signal });
-        return response.ok;
+        for (retries = 0; retries < 5; retries++) {
```

Comment:
We retry 5 times on extension host as well, I think CI can be flaky, hence users machines too could be flaky.